Formula/gauge.rb

```diff
@@ -42,6 +42,12 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
```

Comment:
@sriv I suggest you have the gauge command itself do that as a one-time thing like Homebrew does: https://github.com/Homebrew/brew/blob/021729e1646212d47d3f52c0b29034e7da97693b/Library/Homebrew/cmd/update-report.rb#L22-L47